ibverbs/Cargo.toml

```diff
@@ -20,10 +20,10 @@ license = "MIT OR Apache-2.0"
 
 [dependencies]
 ffi = { path = "../ibverbs-sys", package = "ibverbs-sys", version = "0.3.0" }
-libc = "0.2"
+nix = { version = "0.29", default-features = false, features = ["fs", "poll"] }
 
 [dependencies.serde]
-version = "1.0.100"
+version = "1"
```

Comment:
You'll generally want to keep minor+patch versions in dependencies so that dependents don't accidentally end up with older versions which do not compile with our crate.